README.Rmd

```diff
@@ -5,5 +5,201 @@ output: github_document
 
 [![Build Status](https://travis-ci.org/rstudio/dbtest.svg?branch=master)](https://travis-ci.org/rstudio/dbtest)
 
+```{r echo = FALSE}
+data("testdata", package="dbtest")
+set.seed(229997)
+```
 
+# Overview
 
+`dbtest` uses `testthat` to automate testing of `dbplyr` translations by running the tests against live database connections.  Tests are written in YAML files, and connections are either passed explicitly, read from system DSNs, or configured in YAML files.  Further, it is possible to execute a test suite against multiple database connections.
+
+# Install
+
+To install `dbtest`, you can install the latest version from GitHub:
+
+```{r eval=FALSE}
+remotes::install_github("rstudio/dbtest")
```

Comment:
Not sure how I feel about using `remotes` instead of `devtools`, since most people will already have the latter installed

Reply:
Yeah, that's a good call.  I figured that we would be moving people to `remotes` over time... since it installs almost instantly, is what `devtools` depends on, and encapsulates all of the necessary functionality.

Reply:
What if we added the classic:
```r
if (!"remotes" %in% available.packages())
  install.packages("remotes")
```

Reply:
@edgararuiz thoughts on if I should add that classic line to the README?

```
if (!"remotes" %in% available.packages())
  install.packages("remotes")
```

Reply:
Confirmed with Hadley and Jenny that `devtools` is recommended, as you suggested!  I prefer `remotes` so much because it installs so fast on a clean system :joy: